src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -1027,70 +1026,37 @@ export class JailAddComponent implements OnInit, AfterViewInit {
     // get jail templates as release options
     this.jailService.getTemplates().subscribe(
       (res) => {
-        if (res.result) {
-          for (const i in res.result) {
-            this.template_list.push(res.result[i][1]);
-            this.releaseField.options.push({ label: res.result[i][1] + '(template)', value: res.result[i][1] });
-          }
-        }
-        if (res.error) {
-          this.dialogService.errorReport(T('Error: Displaying templates failed.'), res.error, res.exception);
+        for (let i = 0; i < res.length; i++) {
```

Comment:
`for (const template of templates) {...}` is easier to read imo. not a dealbreaker though :wink:

Reply:
Good suggestion! I will change it :) thanks